research-hub-web/test.sh

```diff
@@ -51,7 +51,7 @@ run_protractor() {
         fi
     else
         echo "Running Local Selenium tests against host: $1"
-        ng e2e
+        npx ng e2e
```

Comment:
Why use `npx` here? The `ng` binary should've already been downloaded from the build steps? Or if the `ng` command isn't being added to the PATH in Jenkins for some reason(?) `npm run e2e` should still work?

Reply:
I changed this because when I tried to run this script locally it would complain that it couldn't find ng. I thought it was best practice to set up dev dependencies like this on a per-project basis. However seems like the [official Angular Getting Started guide](https://angular.io/guide/setup-local) recommends installing the ng binary globally, so I will change this back to the original!